client/genericclient/stream.go

```diff
@@ -31,39 +31,28 @@ import (
 	"github.com/cloudwego/kitex/pkg/streaming"
 )
 
-// NOTE: this is a temporary adjustment for ci check. remove it after fully completing the generic streaming support
```

Comment:
The streaming APIs are exposed now, but will be reverted back to internal just before merging.